app/src/main/java/fr/neamar/kiss/MainActivity.java

```diff
@@ -613,18 +613,11 @@ public void onLauncherButtonClicked(View launcherButton) {
     public boolean dispatchTouchEvent(MotionEvent ev) {
         if (ev.getAction() == MotionEvent.ACTION_DOWN) {
             View v = getCurrentFocus();
-            if (v instanceof SearchEditText) {
+            if (v != null && v instanceof SearchEditText) {
```

Comment:
This check should not be necessary, `instanceof` returns always false if `v` is null.

Reply:
Thanks, fixed.